ocs_ci/helpers/managed_services.py

```diff
@@ -219,3 +222,116 @@ def verify_osd_distribution_on_provider():
         assert (
             osd_count == int(size) / 4
         ), f"Zone {zone} does not have {size/4} osd, but {osd_count}"
+
+
+def verify_storageclient(
+    storageclient_name=None, namespace=None, provider_name=None, verify_sc=True
+):
+    """
+    Verify status, values and resources related to a storageclient
+
+    Args:
+        storageclient_name (str): Name of the storageclient to be verified. If the name is not given, it will be
+            assumed that only one storageclient is present in the cluster.
+        namespace (str): Namespace where the storageclient is present.
+            Default value will be taken from ENV_DATA["cluster_namespace"]
+        provider_name (str): Name of the provider cluster to which the storageclient is connected.
+        verify_sc (bool): True to verify the storageclassclaims and storageclasses associated with the storageclient.
+
+    """
+    storageclient_obj = OCP(
+        kind=constants.STORAGECLIENT,
+        namespace=namespace or config.ENV_DATA["cluster_namespace"],
+    )
+    storageclient = (
+        storageclient_obj.get(resource_name=storageclient_name)
+        if storageclient_name
+        else storageclient_obj.get()["items"][0]
+    )
+    storageclient_name = storageclient["metadata"]["name"]
+    provider_name = provider_name or config.ENV_DATA.get("provider_name", "")
+    endpont_actual = get_storage_provider_endpoint(provider_name)
+    assert storageclient["spec"]["storageProviderEndpoint"] == endpont_actual, (
+        f"The value of storageProviderEndpoint is not correct in the storageclient {storageclient['metadata']['name']}."
+        f" Value in storageclient is {storageclient['spec']['storageProviderEndpoint']}. "
+        f"Value in the provider cluster {provider_name} is {endpont_actual}"
+    )
+    log.info(
+        f"Verified the storageProviderEndpoint value in the storageclient {storageclient_name}"
+    )
+
+    # Verify storageclient status
+    assert storageclient["status"]["phase"] == "Connected"
+    log.info(f"Storageclient {storageclient_name} is Connected.")
+
+    if verify_sc:
+        # Verify storageclassclaims and the presence of storageclasses
+        verify_storageclient_storageclass_claims(storageclient_name)
+        log.info(
+            f"Verified the status of the storageclassclaims associated with the storageclient {storageclient_name}"
+        )
+
+
+def get_storageclassclaims_of_storageclient(storageclient_name):
+    """
+    Get all storageclassclaims associated with a storageclient
+
+    Args:
+        storageclient_name (str): Name of the storageclient
+
+    Returns:
+         List: OCS objects of kind Storageclassclaim
+
+    """
+    sc_claims = get_all_storageclassclaims()
+    return [
+        sc_claim
+        for sc_claim in sc_claims
+        if sc_claim.data["spec"]["storageClient"]["name"] == storageclient_name
+    ]
+
+
+def get_all_storageclassclaims():
```

Comment:
This can be moved to the `storage_cluster` file https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/storage_cluster.py.

Reply:
Storageclassclaims is also used in managed services clusters only.